Lib/fontgoggles/mac/mainWindow.py

```diff
@@ -254,12 +259,13 @@ def setupFontListGroup(self):
             dict(view=self.compileOutput, identifier="compileOutput", canCollapse=True,
                  size=compileOutputSize, minSize=30, resizeFlexibility=False),
         ]
-        self.fontListSplitView = MySplitView((0, 40, 0, 0), paneDescriptors, dividerStyle="thin",
+        self.fontListSplitView = MySplitView((0, 41, 0, 0), paneDescriptors, dividerStyle="thin",
                                              isVertical=False)
         if not self.project.uiSettings.compileOutputVisible:
             self.fontListSplitView.togglePane("compileOutput")
 
         group.textEntry = self.textEntry
+        group.divider = self._divider
```

Comment:
Please construct the `HorizontalLine` here, inline. I only add vanilla objects to self if they need to be referenced elsewhere.

Reply:
Sure.
Normally I would make the scroll view one pixel bigger to hide its border and only leave the top border visible. But I don’t know how to do that in vanilla.